invite.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Home
+---
+<div class="row flow-text">
```

Comment:
This should use the meta tag, or preferably this plugin https://github.com/jekyll/jekyll-redirect-from

Reply:
Meta tag is for the head which I can't add so Ill check the plugin